AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/work-allocation.component.html

```diff
@@ -1 +1,137 @@
-<p>work-allocation works!</p>
+<div class="govuk-!-margin-top-7">
+  <details *ngIf="isVhTeamLeader" class="govuk-details" data-module="govuk-details">
+    <summary class="govuk-details__summary" tabindex="0" role="button">
+      <span class="govuk-details__summary-text" id="citizen-contact-details"> Upload working hours / non-availability </span>
+    </summary>
+    <div class="govuk-!-margin-left-5">
+      <div class="govuk-grid-row">
+        <div class="govuk-grid-column-one-third">
+          <p>Upload working hours:</p>
+        </div>
+        <div class="govuk-grid-column-two-thirds govuk-!-margin-top-2">
+          <label class="govuk-label govuk-!-font-size-16" for="working-hours-file-upload"> Select .CSV file to upload </label>
+          <input class="govuk-file-upload" id="working-hours-file-upload" name="working-hours-file-upload" type="file" />
+        </div>
+      </div>
+      <div class="govuk-grid-row">
+        <div class="govuk-grid-column-one-third">
+          <p>Upload non-availability hours:</p>
+        </div>
+        <div class="govuk-grid-column-two-thirds govuk-!-margin-top-2">
+          <label class="govuk-label govuk-!-font-size-16" for="non-availability-hours-file-upload"> Select .CSV file to upload </label>
+          <input class="govuk-file-upload" id="non-availability-hours-file-upload" name="non-availability-hours-file-upload" type="file" />
+        </div>
+      </div>
+    </div>
+  </details>
+  <details class="govuk-details" data-module="govuk-details">
```

Comment:
Would this need a `*ngIf="isVhTeamLeader"` too?

Reply:
Hi Oliver. This one does not as this is accessible to both officers and team leaders, unlike the other 3 which only team leads can access.